src/labels/CustomLabels.labels

```diff
@@ -1913,6 +1913,13 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>bdiAccountCustomIdError</shortDescription>
         <value>The Account Custom Unique ID setting field {0} must have matching fields in the NPSP Data Import object whose API Names are {1} and {2}.</value>
     </labels>
+    <labels>
+        <fullName>bdiAdditionalObjChildNotUpdated</fullName>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>bdiAdditionalObjChildNotUpdated</shortDescription>
+        <value>Error when updating child record lookup field(s) with new parent record id. Error message is:</value>
```

Comment:
```suggestion
        <value>Unable to update the child record's lookup field with new parent record's ID. It was unable to update because of this system error:</value>
```